pseudocode.txt

```diff
@@ -0,0 +1,49 @@
+# standardize the data format
+def changeFileExtensions(folderPath, newFileType):
```

Comment:
Changing file extensions does not change the type of the file. You need to rewrite them if you want to e.g. transform tab-separated to comma-separated

Reply:
Oh, so this code does not work for other files! The code works to open the .flm files on the CfA Supernova Data Archive, so I will mention that in the documentation. I will write code to deal with other types of data later if time allows!